components/blocks/FilesToCards.vue

```diff
@@ -218,13 +240,18 @@ export default {
     ascending: true,
     sortBy: [],
     searchGroup: [],
+    tagColors: {'tags': 'is-link', 'groups': 'is-yellow', 'languages': 'is-info'}
   }),
   computed: {
     cardTags() {
-      const tags = this.data.map((card) => card.group);
-      return tags
-        .filter((group, index) => tags.indexOf(group) === index)
-        .sort();
+      let tags = this.data.map((card) => card.tags);
+      const groups = this.data.map((card) => card.groups);
+      const languages = this.data.map((card) => card.languages);
+      tags = tags
+        .concat(groups, languages)
+        .flat()
+        .filter((e) => e);
```

Comment:
```suggestion
      const tags = ['tags', 'groups', 'languages'].map((tagType) => this.data.map((card) => card[tagType])).flat();
```
I'm not clear on what the filter statement is for on 253

Reply:
it's for filtering out the empty string that sometimes appears from the yaml :(

Reply:
the suggestion doesn't give the desired result. it returns a list of arrays, some duplicates, and some with multiple array items. it should return a unique list of unique tags, groups, and languages.

Reply:
I think you need to add `.filter((tag, index) => tags.indexOf(tag) === index).sort()` at the end - looks like that was what was uniquing it